src/contexts/__tests__/useSdkOptions.tsx

```diff
@@ -20,30 +46,40 @@ describe('context', () => {
     })
 
     it('gets correct options data', () => {
-      const fakeOptions: NarrowSdkOptions = {
-        token: 'fake-sdk-token',
-        containerId: 'onfido-mount',
-        language: 'fr',
-        userDetails: {
-          smsNumber: '+447495023357',
-        },
-        steps: [
-          { type: 'welcome' },
-          { type: 'document' },
-          { type: 'face' },
-          { type: 'complete' },
-        ],
-        events: new EventEmitter2(),
-      }
-
       const wrapper = mount(
-        <SdkOptionsProvider options={fakeOptions}>
+        <SdkOptionsProvider options={defaultOptions}>
           <DummyComponent />
         </SdkOptionsProvider>
       )
 
       const span = wrapper.find('DummyComponent > span')
-      expect(span.prop('data-sdk-options')).toMatchObject(fakeOptions)
+      expect(span.prop('data-sdk-options')).toMatchObject(defaultOptions)
+      expect(span.prop('data-step')).toBeUndefined()
+    })
+
+    it('gets correct step config', () => {
+      const wrapper = mount(
+        <SdkOptionsProvider options={defaultOptions}>
+          <DummyComponent step="document" />
```

Comment:
Just to iterate on my previous comment, wouldn't it make more sense for each component to receive its own props rather than receive its "step" name? 🤔

Reply:
At which point in the code do we define `step`?

Reply:
Well the `DummyComponent` is just a _dummy_ one, set up as a pure environment for `useSdkOptions()` testing purposes only. In real situations, developers will decide which `step` the component is being implemented for and call `useSdkOptions()`'s `findStep` against that `step`. For instance, if I want to implement a new `DocumentVideo` step component, then I know that the component wants to know about `document` step, and I'll try to `findStep('document')`. There're no such `step` props passed down in real life.